configs/components/puppet.rb

```diff
@@ -1,6 +1,10 @@
 component "puppet" do |pkg, settings, platform|
   pkg.load_from_json("configs/components/puppet.json")
 
+  if platform.is_windows?
+    pkg.add_source("file://resources/files/windows/environment.bat", sum: "140cc75b2dabfb015468ecdafc533090")
```

Comment:
I would rather install this file as a part of the puppet-agent project file, but I wasn't sure if I could do that. This was the next best option that I could think of.